src/popup/router/components/AccountInfo.vue

```diff
@@ -60,15 +70,13 @@ export default {
   },
   mixins: [CopyMixin],
   props: {
-    accountIdx: { type: Number, default: -1 },
     color: { type: String, default: 'black' },
-    showCopyIcon: { type: Boolean, default: false },
+    canCopyAddress: { type: Boolean, default: false },
```

Comment:
As the default state of the Boolean props is `false` this can be shortened to `canCopyAddress: Boolean`.